### PR TITLE
Gumbo

### PR DESCRIPTION
**Pull request by isaachier**

### CODE REVIEW

**Comment by ruslo** [comment]
Have you tested it? I expect pull request to `pkg.template` first. See docs:
* https://docs.hunter.sh/en/latest/creating-new/create/cmake.html#pull-requests-tests

**Comment by isaachier** [comment]
OK good to know. This was more of a placeholder. Need to fix some of the toolchains.

File: docs/packages/pkg/gumbo.rst [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+.. _pkg.gumbo:
+
+gumbo
+=====
+
+-  `Official <https://github.com/gumbo/gumbo>`__
+-  `Hunterized <https://github.com/isaachier/gumbo>`__
+-  `Example <https://github.com/ruslo/hunter/blob/master/examples/gumbo/CMakeLists.txt>`__
+-  Added by `Isaac Hier <https://github.com/isaachier>`__ (`pr-N <https://github.com/ruslo/hunter/pull/N>`__)
```
Comment (ruslo) [inline docs/packages/pkg/gumbo.rst]: Pull request number:
* https://docs.hunter.sh/en/latest/creating-new/create/cmake.html#pull-requests

**Comment by isaachier** [comment]
Please create a fork of https://github.com/google/gumbo-parser in hunter-packages.

**Comment by ruslo** [comment]
> Please create a fork of https://github.com/google/gumbo-parser in hunter-packages.

Done: https://github.com/hunter-packages/gumbo-parser

**Comment by ruslo** [comment]
Testing:
* https://ci.appveyor.com/project/ingenue/hunter/build/1.0.2466
* https://travis-ci.org/ingenue/hunter/builds/280058879?utm_source=github_status&utm_medium=notification

**Comment by isaachier** [comment]
Resolves #767.

**Comment by ruslo** [comment]
Testing:
* https://ci.appveyor.com/project/ingenue/hunter/build/1.0.2467
* https://travis-ci.org/ingenue/hunter/builds/280143168?utm_source=github_status&utm_medium=notification

**Comment by ruslo** [comment]
Uploading:
* https://ci.appveyor.com/project/ingenue/hunter/build/1.0.2468
* https://travis-ci.org/ingenue/hunter/builds/280324184?utm_source=github_status&utm_medium=notification

**Comment by ruslo** [comment]
Released: https://github.com/ruslo/hunter/releases/tag/v0.19.112

**Pull request merged**